python/lsst/meas/extensions/piff/piffPsfDeterminer.py

```diff
@@ -105,6 +105,10 @@ class PiffPsfDeterminerConfig(BasePsfDeterminerTask.ConfigClass):
         check=_validateGalsimInterpolant,
         default="Lanczos(11)",
     )
+    _debugStarData = pexConfig.Field[bool](
```

Comment:
I have a strong preference for not making private configs a new thing in the stack.    `debugStarData` would be fine.